src/deploy/osp_deployer/settings/sample_hci.properties

```diff
@@ -24,13 +21,14 @@
         "is_director": "true",
         "hostname": "director",
         "root_password": "xxxxxxxxxx",
-
         "provisioning_ip": "192.168.120.13",
-
         "management_ip":"192.168.110.13",
         "public_api_ip":"192.168.190.13",
-
-        "private_api_ip":"192.168.140.13"
+        "private_api_ip":"192.168.140.13",
+        "public_api_if": "enp1s0",
+        "provisioning_if": "enp2s0",
+        "management_if": "enp3s0",
+        "private_api_if": "enp4s0"
```

Comment:
I assume those are used for setting up routing etc, could we not do this against the bonds rather than interfaces ?

Reply:
Unfortunately you have to restart the physical interfaces for routes to take on Director VM, tried avoiding this but restarting bond, bridge and routes has no effects on routes being functional, they will show up in "ip route" but are non-functional until you ifdown/ifup the interface (enp2s0 in the case of provisioning network for example)

Reply:
Could we grab the interfaces name from the .properties to avoid additional/duplicate settings in ini ?

Reply:
Sorry i was thinking this was for the SAH interfaces. If tis is director vm, let s not make them a settings, it s a pre set list of interfaces that ll always be enp1s0...enp4s0, should not be "configurable"

Reply:
Will move to constants and out of ini.

Reply:
Done